pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -489,4 +491,21 @@ type (
 		// +optional
 		PublicKeyName *string `json:"publicKeyName,omitempty"`
 	}
+
+	// MixedInstances holds the configuration for spot instances
+	// TODO: validation of base capcity : make required
+	// TODO allocation strategy?
```

Comment:
there are a few fields that only have one value, we don't need to include those right now

Reply:
:+1: I didn't include those